edalize/incisive.py

```diff
@@ -0,0 +1,64 @@
+import os
+import logging
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+
+class Incisive(Edatool):
+
+    _description = """ Cadence Incisive Verilog Backend
+
+Example snippet of a CAPI2 description file for Incisive:
+
+.. code:: yaml
+
+   incisive:
+     incisive_options:
+       # Compile-time options passed to the incisive command
+       - -some_buildarg
+     run_options:
+       # Run-time options passed to the simulation itself
+       - +some+plusarg
```

Comment:
Is plusarg really a good example here? Shouldn't they rather be handled with parameters of type plusarg instead? Not a dealbreaker but could avoid some confusion

Reply:
Yeah I'll read some more of the docs here to figure out an actual valid runarg